usb-modem-drivers/pkg.yaml

```diff
@@ -0,0 +1,27 @@
+name: usb-modem-drivers-pkg
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+    runtime: true
+  - stage: kernel
+steps:
+  - env:
+      ARCH: {{ if eq .ARCH "aarch64"}}arm64{{ else if eq .ARCH "x86_64" }}x86_64{{ else }}unsupported{{ end }}
+    install:
+      - |
+        export KERNELRELEASE="{{ .linux_version }}-talos"
+
+        mkdir -p /rootfs/lib/modules/${KERNELRELEASE}/kernel/drivers/{usb,net}
+
+        cp -r /lib/modules/${KERNELRELEASE}/kernel/drivers/usb/ /rootfs/lib/modules/${KERNELRELEASE}/kernel/drivers/
```

Comment:
if we enable more `usb` drivers, this might copy more drivers than just usb modem drivers?

Reply:
yes, it will, but otherwise would have to optionally keep a list of the modules names:

```bash
_out/lib/modules/6.1.32-talos/
├── kernel
│   ├── drivers
│   │   ├── infiniband
│   │   │   └── sw
│   │   │       └── rxe
│   │   │           └── rdma_rxe.ko
│   │   ├── net
│   │   │   ├── ethernet
│   │   │   │   └── mellanox
│   │   │   │       ├── mlx4
│   │   │   │       │   ├── mlx4_core.ko
│   │   │   │       │   └── mlx4_en.ko
│   │   │   │       ├── mlx5
│   │   │   │       │   └── core
│   │   │   │       │       └── mlx5_core.ko
│   │   │   │       ├── mlxfw
│   │   │   │       │   └── mlxfw.ko
│   │   │   │       └── mlxsw
│   │   │   │           ├── mlxsw_core.ko
│   │   │   │           ├── mlxsw_i2c.ko
│   │   │   │           ├── mlxsw_minimal.ko
│   │   │   │           ├── mlxsw_pci.ko
│   │   │   │           └── mlxsw_spectrum.ko
│   │   │   ├── phy
│   │   │   │   ├── ax88796b.ko
│   │   │   │   └── smsc.ko
│   │   │   └── usb
│   │   │       ├── aqc111.ko
│   │   │       ├── asix.ko
│   │   │       ├── ax88179_178a.ko
│   │   │       ├── cdc_eem.ko
│   │   │       ├── cdc_ether.ko
│   │   │       ├── cdc_mbim.ko
│   │   │       ├── cdc_ncm.ko
│   │   │       ├── cdc-phonet.ko
│   │   │       ├── cdc_subset.ko
│   │   │       ├── cx82310_eth.ko
│   │   │       ├── dm9601.ko
│   │   │       ├── gl620a.ko
│   │   │       ├── huawei_cdc_ncm.ko
│   │   │       ├── int51x1.ko
│   │   │       ├── kalmia.ko
│   │   │       ├── lg-vl600.ko
│   │   │       ├── mcs7830.ko
│   │   │       ├── net1080.ko
│   │   │       ├── plusb.ko
│   │   │       ├── qmi_wwan.ko
│   │   │       ├── r8153_ecm.ko
│   │   │       ├── rndis_host.ko
│   │   │       ├── sierra_net.ko
│   │   │       ├── smsc75xx.ko
│   │   │       ├── smsc95xx.ko
│   │   │       ├── sr9700.ko
│   │   │       ├── sr9800.ko
│   │   │       ├── usbnet.ko
│   │   │       └── zaurus.ko
│   │   ├── usb
│   │   │   ├── class
│   │   │   │   └── cdc-wdm.ko
│   │   │   └── serial
│   │   │       ├── option.ko
│   │   │       └── usb_wwan.ko
│   │   └── virtio
│   │       ├── virtio_balloon.ko
│   │       ├── virtio_input.ko
│   │       ├── virtio_mmio.ko
│   │       ├── virtio_pci.ko
│   │       ├── virtio_pci_legacy_dev.ko
│   │       └── virtio_pci_modern_dev.ko
│   └── lib
│       ├── objagg.ko
│       └── parman.ko
```